users/README.md

```diff
@@ -34,6 +34,12 @@
 		"super_user": "boolean",
 		"in_discord": "boolean"
 	},
+    "archivedDetails": {
```

Comment:
what will happen if user is un archived. Will this object be deleted then because for an un archived user this object wont make sense.

Reply:
If the user chooses to unarchive, the previous data will persist in the database. If the user is archived again, the archivedDetails object will be updated to reflect the new reason and date. Storing this object in the database allows us to track the previous reasons and dates when the user was archived. So keeping this model in the user modal.